docker-compose.yaml

```diff
@@ -1,5 +1,72 @@
----
-version: '3.0'
+version: '3.8'
```

Comment:
As long as these changes still work, let's leave this at `3.0`.

Reply:
sure, it seems furthermore that version is itself obsolete

```
$ docker compose --profile litestream up
WARN[0000] /home/ubuntu/ar-io-node/docker-compose.yaml: `version` is obsolete
```